pkg/didcomm/protocol/introduce/models.go

```diff
@@ -64,7 +64,8 @@ type Content struct {
 // PleaseIntroduceTo includes all field from To structure
 // also it has Discovered the field which should be provided by help-me-discover protocol
 type PleaseIntroduceTo struct {
-	To
+	// nolint: staticcheck
```

Comment:
what was the linter issue?

Reply:
JSON does not have `squash` e.g `json:",squash"`
Squash needs to be used when we inherit a type.